Monika After Story/game/script-topics.rpy

```diff
@@ -15749,3 +15749,54 @@ label monika_literature_value:
     m 1eku "Honestly, I think if more people valued books and poems a little more, the world would be a much better place."
     m 1hksdlb "That's just my opinion as president of a literature club, though. {w=0.2}I guess most people wouldn't think that deeply about it."
     return
+    
+# Do you like nature?
+default persistent._mas_pm_likes_nature = None
+
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="monika_nature",category=['nature', 'you'],prompt="The outdoors",random=True))
+ 
+label monika_nature:
+    m 2esd "Hey, [player]..."
+    $ _history_list.pop()
+    menu:
+        m "Do you like nature{fast}?"
+        "Yes.":
+            $ persistent._mas_pm_likes_nature = True
+            m 2sub "Really? That's wonderful!"
+            m 3wud "You know, I think nature is something we should cherish."
```

Comment:
![1](https://user-images.githubusercontent.com/45746485/93428218-5a8f3c80-f8bf-11ea-9574-6a761b65e56c.png)

I think 3eub seems more natural in this instance. Sentence looks fine.

Reply:
Open smile is probably better here, but I'd hesitate to go directly into regular eyes from sparkle eyes since it feels like too much of a jump. If at some point we end up going with something like 1hua for the first sentence, then 3eub would be a great followup to that.

Reply:
Agreed, I would not want it as a followup to the sparkle eyes.